### ISSUE TITLE
Question: Apply CSS to and svg while zooming

### ISSUE THREAD

**Issue by CoderGarm** [svgdotjs/svg.panzoom.js#100]
Good morning,

I have and svg and some strokes have the following css rule applied.

`stroke-width: 0.1%`

The rule will be rendered correctly - but not while zooming or after the zoom ends.

It will be applied afterwards only when I drag the svg.
Let me describe it somehow:
You have the svg and zoom in, the strokes will be bigger and bigger. You stop zooming - nothing happends.
Pan the canvas a tick and the strokes' width is correct.

Do someone have an idea how to "apply" the css while zooming?

I changed it from setting the stroke-width programatically which worked good - but with "unnecessary" code in the case that it could be done by css.

**Comment by Fuzzyma** [comment]
I think the reason for this is browser optimization. You can try the css rule will-change. Maybe it makes a difference

**Comment by CoderGarm** [comment]
Hey @Fuzzyma,

thanks for that answer, 
I tried this before and now again but it didnt work. 

I tried instead a different approach which seems unintuitive:

`stroke-width: 0.5px;
    vector-effect: non-scaling-stroke;`

It seems that vector-effect do exactly what I wanted to achieve.  

So instead binding the stroke-width to the viewport by the relative argument it is fixed but ignores any zooming effect.
